resources/i18n/fr.json5

```diff
@@ -1,3220 +1,1596 @@
-  // TODO New key - Add a translation
-  "item.edit.item-mapper.buttons.remove": "Remove item's mapping for selected collections",
-  
+  "item.edit.item-mapper.buttons.remove": "Supprimer l'association d'Item pour les Collections sélectionnées",
   // "item.edit.item-mapper.cancel": "Cancel",
-  // TODO New key - Add a translation
-  "item.edit.item-mapper.cancel": "Cancel",
-  
+  "item.edit.item-mapper.cancel": "Annuler",
   // "item.edit.item-mapper.description": "This is the item mapper tool that allows administrators to map this item to other collections. You can search for collections and map them, or browse the list of collections the item is currently mapped to.",
-  // TODO New key - Add a translation
-  "item.edit.item-mapper.description": "This is the item mapper tool that allows administrators to map this item to other collections. You can search for collections and map them, or browse the list of collections the item is currently mapped to.",
-  
+  "item.edit.item-mapper.description": "L'Item Mapper permet aux administrateurs de Collection d'associer des Items avec d'autres Collections. Vous pouvez rechercher des Collections et les associer ou parcourir la liste des Collections auxquelles l'Item est actuellement associé.",
   // "item.edit.item-mapper.head": "Item Mapper - Map Item to Collections",
-  // TODO New key - Add a translation
-  "item.edit.item-mapper.head": "Item Mapper - Map Item to Collections",
-  
+  "item.edit.item-mapper.head": "Item Mapper - Associer Item avec Collections",
   // "item.edit.item-mapper.item": "Item: \"<b>{{name}}</b>\"",
-  // TODO New key - Add a translation
-  "item.edit.item-mapper.item": "Item: \"<b>{{name}}</b>\"",
-  
+  "item.edit.item-mapper.item": "Item : \"<b>{{name}}</b>\"",
   // "item.edit.item-mapper.no-search": "Please enter a query to search",
-  // TODO New key - Add a translation
-  "item.edit.item-mapper.no-search": "Please enter a query to search",
-  
+  "item.edit.item-mapper.no-search": "Veuillez entrer un élément à rechercher",
   // "item.edit.item-mapper.notifications.add.error.content": "Errors occurred for mapping of item to {{amount}} collections.",
-  // TODO New key - Add a translation
-  "item.edit.item-mapper.notifications.add.error.content": "Errors occurred for mapping of item to {{amount}} collections.",
-  
+  "item.edit.item-mapper.notifications.add.error.content": "Des erreurs se sont produites à l'association de l'Item avec {{amount}} Collections.",
   // "item.edit.item-mapper.notifications.add.error.head": "Mapping errors",
-  // TODO New key - Add a translation
-  "item.edit.item-mapper.notifications.add.error.head": "Mapping errors",
-  
+  "item.edit.item-mapper.notifications.add.error.head": "Erreurs d'association",
   // "item.edit.item-mapper.notifications.add.success.content": "Successfully mapped item to {{amount}} collections.",
-  // TODO New key - Add a translation
-  "item.edit.item-
```

Comment:
Comment: For localized version (at least) I will use both masculine and feminine forms : "Nouvel utilisateur ? Nouvelle utilisatrice ? Cliquer ici pour vous enregistrer"

Reply:
I have used another translation here to keep it simple and avoid the gender discussion for this label: 
"Pas encore de compte ? Cliquez ici pour vous enregistrer."

I'll discuss feminisation of other labels with other (female) library managers in France because I think feminisation is much more common in Canada and I'm not too sure that for example "Auteure" is already commonly used here.
Also feminisation can have an impact on the interface especially when it comes to menu entries such as browse indexes.